### PR TITLE
Clarify description of "logging.file.max-history"

### PR DESCRIPTION
**Pull request by lihongmingming**
fix the #17566

### CODE REVIEW

**Comment by pivotal-issuemaster** [comment]
@lihongmingming Please sign the [Contributor License Agreement](https://cla.pivotal.io/sign/spring?repositoryId=spring-projects/spring-boot&pullRequestId=17567)!

[Click here](https://cla.pivotal.io/sync/spring?repositoryId=spring-projects/spring-boot&pullRequestId=17567) to manually synchronize the status of this Pull Request.

See the [FAQ](https://cla.pivotal.io/about) for frequently asked questions.

**Comment by pivotal-issuemaster** [comment]
@lihongmingming Thank you for signing the [Contributor License Agreement](https://cla.pivotal.io/sign/spring?repositoryId=spring-projects/spring-boot&pullRequestId=17567)!

**Comment by bclozel** [comment]
@lihongmingming Could you update your PR with something along the lines of:

> Maximum number of days worth of log history. Only supported with the default logback setup.

File: spring-boot-project/spring-boot/src/main/resources/META-INF/additional-spring-configuration-metadata.json [hunk 1]
```diff
@@ -105,7 +105,7 @@
     {
       "name": "logging.file.max-history",
       "type": "java.lang.Integer",
-      "description": "Maximum of archive log files to keep. Only supported with the default logback setup.",
+      "description": "Maximum of retention days the log archive log files to keep. Only supported with the default logback setup.",
```
Comment (prodingerd) [inline spring-boot-project/spring-boot/src/main/resources/META-INF/additional-spring-configuration-metadata.json]: What about using this description?

```suggestion
      "description": "Maximum number of days, archive log files are kept. Only supported with the default logback setup.",
```

**Comment by prodingerd** [comment]
Why are you closing the PR @lihongmingming?

**Comment by snicoll** [comment]
@lihongmingming thank you for making your first contribution to Spring Boot. Your contribution has been merged in `2.1.x` and `master`.

**Pull request closed without merging**